chef_master/source/data_collection_habitat.rst

```diff
@@ -3,4 +3,4 @@ Data Collection with Habitat
 `[edit on GitHub] <https://github.com/chef/chef-web-docs/blob/master/chef_master/source/data_collection_habitat.rst>`__
 
 
-The data collection capabilities of Chef Automate can also be used to collect and report on Habitat ring data. The Prism Habitat package collects this data and sends it to a Chef Automate server's REST API endpoint. You can configure settings like the data collector URL, token, the Habitat supervisor used to get the ring information, and so on. For more information on the Prism package, see :doc:`Habitat Prism </habitat_prism>`. For more information on Habitat, see the `Habitat site <https://habitat.sh/>`__.
+The data collection capabilities of Chef Automate can also be used to collect and report on Habitat ring data. The Prism Habitat package collects this data and sends it to a Chef Automate server's REST API endpoint. You can configure settings like the data collector URL, token, the Habitat supervisor used to get the ring information, and so on. For more information on the Prism package, see `Habitat Prism </habitat_prism.html>`__. For more information on Habitat, see the `Habitat site <https://habitat.sh/.html>`__.
```

Comment:
Because this topic is removed in a subsequent PR, make sure you merge this one in first.

Reply:
yep, holding off on the habitat one till this goes through.